docs/android/emulator/device-manager.md

```diff
@@ -0,0 +1,434 @@
+
+- **Duplicate and Edit** &ndash; Duplicates the currently-selected device and opens it in the **New Device** screen with a new name that's similar to the existing device. For example, selecting **Pixel 2 - API 28** and pressing **Duplicate and Edit** appends a counter to the name: **Pixel 2 - API 28 (1)**.
+
+- **Start with Factory Defaults** &ndash; TODO
+
+- **Start with Kernel Logs** &ndash; TODO
+
+- **Download System Image** &ndash; Downloads the Android OS system image for the device, if it's not already downloaded.
+
+- **Reveal in Explorer** &ndash; Opens Windows Explorer and navigates to the folder that holds the files for the virtual device.
+
+- **Repair** &ndash; TODO
+
+- **Factory Reset** &ndash; Resets the selected device to its default settings, erasing any user changes made to the internal state of the device while it was running. This action also erases the current [Quick Boot](../deployment/debug-on-emulator.md#quick-boot) snapshot if it exists. This change doesn't alter modifications that you make to the virtual device during creation and editing. A dialog box will appear with the reminder that this reset cannot be undone &ndash; press **Factory Reset** to confirm the reset.
+
+- **Delete** &ndash; Permanently deletes the selected virtual device. A dialog box will appear with the reminder that deleting a device cannot be undone. Press **Delete** if you are certain that you want to delete the device.
+
+## Troubleshooting
+
+The following sections explain how to diagnose and work around problems that may occur when using the Android Device Manager to configure virtual devices.
+
+### Android SDK in non-standard location
+
+Typically, the Android SDK is installed at _C:\\Program Files (x86)\\Android\\android-sdk_. If the SDK isn't installed at this location, you may get this error when you launch the Android Device Manager:
+
+:::image type="content" source="media/device-manager/win/29-sdk-error.png" alt-text="Android SDK instance error.":::
+
+To work around this problem, do the following:
+
+01. Open the _%LOCALAPPDATA%\\Xamarin\\Logs\\XamarinDeviceManager_ folder.
+
+    The `%LOCALAPPDATA%` value is an environment variable that typically points to the _C:\\Users\\{your user name}\\AppData\\Local_ folder.
+
+01. Open one of the log files and search for an entry describing **Config file path**. For example:
+
+    :::image type="content" source="media/device-manager/win/31-config-file-path.png" alt-text="Config file path in log file.":::
+
+01. Navigate to this folder and open the _user.config_ in a text editor.
+
+01. In _user.config_, find the `<UserSettings>` element and add an **AndroidSdkPath** attribute to it. Set this attribute to the path where the Android SDK is installed on your computer and save the file. For example, `<UserSettings>` would look like the following if the Android SDK was installed at _C:\\Programs\\Android\\SDK_:
+
+    <!-- TODO: Does visual studio handle this some how now? -->
```

Comment:
I'd be inclined to comment out/remove this section, unless you are sure it works. Using a non-default location is going to open up a world of pain.